vars/Debian.yml

```diff
@@ -1,4 +1,5 @@
 ---
+nginx_conf_path: /etc/nginx/conf.d/*.conf
```

Comment:
I'd rather do a `/etc/nginx/conf.d` (so it's just a path), and then in the template, add the `/*.conf`.